server/datastore/mysql/labels.go

```diff
@@ -282,17 +282,20 @@ func labelDB(ctx context.Context, lid uint, q sqlx.QueryerContext) (*fleet.Label
 
 // ListLabels returns all labels limited or sorted by fleet.ListOptions.
 func (ds *Datastore) ListLabels(ctx context.Context, filter fleet.TeamFilter, opt fleet.ListOptions) ([]*fleet.Label, error) {
+	if opt.After != "" {
+		return nil, &fleet.BadRequestError{Message: "'after' parameter is not supported"}
```

Comment:
Minor suggestion: I'd skip the quotes (at least in recent error messages that reference specific parameters, we did without them. Not sure if that format is super consistent everywhere though.)
```suggestion
		return nil, &fleet.BadRequestError{Message: "after parameter is not supported"}
```